api/src/test/java/org/openmrs/module/openconceptlab/updater/ImporterTest.java

```diff
@@ -20,6 +20,7 @@
 import static org.hamcrest.Matchers.not;
 import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;

```

Comment:
Yes, I did before Paweł but I didn't have time to add coverage and then  Paweł did that. 
I have deleted this project from my codacy account today so it shoudn't add any  comments